Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/screens/animelist/AnimeList.kt

```diff
@@ -1,14 +1,22 @@
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.lifecycle.viewmodel.compose.viewModel
 import com.abhijith.animex.ui.screens.animelist.AnimeListItem
+import com.abhijith.animex.ui.screens.animelist.viewmodel.AnimeListViewModel
 
 @Composable
-fun AnimeList() {
-    val names = List(15) { "$it" }
+fun AnimeList(viewModel: AnimeListViewModel = viewModel()) {
```

Comment:
1. We need to understand what does `= viewModel()` do?
2. If the function recompose, are we creating new instance of ViewModel again and again?